miden-lib/build.rs

```diff
@@ -231,7 +231,7 @@ fn generate_kernel_proc_hash_file(kernel: KernelLibrary) -> Result<()> {
         format!(
             r#"/// This file is generated by build.rs, do not modify
 
-use miden_objects::{{digest, Digest, Felt}};
+use miden_objects::{{digest, Digest}};
```

Comment:
Question: why do we need double curly brackets here? Would something like this not work?
```rust
use miden_objects::{digest, Digest};
```

Reply:
The double curly is how you [escape them](https://doc.rust-lang.org/std/fmt/index.html#escaping) in rust. Required since this is writing/formatting the text and `{}` is the formatting syntax.

Reply:
Ah! I missed that.